clients/dos/src/main/kotlin/DosModel.kt

```diff
@@ -46,6 +46,15 @@ data class UploadUrlResponseBody(
     val message: String? = null
 )
 
+/** Information needed to create a Package row in DOS database */
```

Comment:
To my understanding instances currently are used solely as parameter for a request to create a scan job.
I do not understand why this comment talks about inserting a row into a database.

Reply:
Yes, maybe this comment went a little too much into the inner workings of DOS server, I wonder, should I omit this comment completely, or change it to something like "Information needed about the package on DOS server"?